lnbits/wallets/zbd.py

```diff
@@ -0,0 +1,171 @@
+import asyncio
+from typing import AsyncGenerator, Dict, Optional
+
+import httpx
+from loguru import logger
+
+from lnbits.settings import settings
+
+from .base import (
+    InvoiceResponse,
+    PaymentResponse,
+    PaymentStatus,
+    StatusResponse,
+    Unsupported,
+    Wallet,
+)
+
+
+class ZBDWallet(Wallet):
+    """https://zbd.dev/api-reference/"""
+
+    def __init__(self):
+        if not settings.zbd_api_endpoint:
+            raise ValueError("cannot initialize ZBDWallet: missing zbd_api_endpoint")
+        if not settings.zbd_api_key:
+            raise ValueError("cannot initialize ZBDWallet: missing zbd_api_key")
+
+        self.endpoint = self.normalize_endpoint(settings.zbd_api_endpoint)
+        self.auth = {
```

Comment:
- any reason to make `auth `a field on this class? I do not see `self.auth` referenced anywhere else.
- nitpick: rename to `headers = { ...`